lib/oli/delivery/sections.ex

```diff
@@ -57,6 +57,7 @@ defmodule Oli.Delivery.Sections do
   alias Oli.Delivery.Paywall
   alias Oli.Delivery.Sections.PostProcessing
   alias Oli.Branding.CustomLabels
+  alias OliWeb.Router.Helpers, as: Routes
```

Comment:
We should not have an OliWeb dependency in Oli code.

Reply:
Thanks for the feedback, removed it as it is no longer necessary